script.js

```diff
@@ -0,0 +1,123 @@
+//Board kept as array of 9 elements either undefined, "X" or "O"
+var board = new Array(9);
+//Turn tracker
+var xTurn = true;
+
+//Board squares and restart button
+var pos0 = document.getElementById('pos0');
+var pos1 = document.getElementById('pos1');
+var pos2 = document.getElementById('pos2');
+var pos3 = document.getElementById('pos3');
+var pos4 = document.getElementById('pos4');
+var pos5 = document.getElementById('pos5');
+var pos6 = document.getElementById('pos6');
+var pos7 = document.getElementById('pos7');
+var pos8 = document.getElementById('pos8');
```

Comment:
it looks like each one of these has the class `space`. what if you `document.querySelectorAll(".space")` and loop through them?